docs/getting-started/creating-a-new-application.md

```diff
@@ -4,4 +4,43 @@ title: Creating a new application
 sidebar_label: Create a new application
 ---
 
-## TODO
\ No newline at end of file
+## Prerequisites
+
+Before you start, check to make sure you have the following installed:
+
+- [Node.js](https://nodejs.org/en/) v10.15.x or later
+- [npm](https://www.npmjs.com/) v5.x or later
+
+## Create your application
+
+[Create GraphQL](https://github.com/aerogear/create-graphql) is a command-line utility for creating Graphback applications.
+
+Installing with npm:
+
+```sh
+npm install -g create-graphql
```

Comment:
`npm init graphql projectname` - that is the whole purpose of the `create-x`
Alternatively `npx create-graphql project

Reply:
+1 for the first proposition. 

The alternative could be added, with something like `or if you have npx installed, you can start a new project with...`

Reply:
In a similar reason as https://github.com/aerogear/graphback/pull/1636#discussion_r448947424

npx users know how to use npx, so why focus on documenting with npx when they know how to do both, but non npx people do not?

Reply:
npx is embeedded into node.js :P

Reply:
> npx users know how to use npx, so why focus on documenting with npx when they know how to do both, but non npx people do not?

Makes sense.

Reply:
Since npx is embedded in node.js and does not require anything extra installed, I will use it in the example.